recipes/pixman/all/conanfile.py

```diff
@@ -91,11 +96,11 @@ def package(self):
         else:
             autotools = self._configure_autotools()
             autotools.install()
-        la = os.path.join(self.package_folder, "lib", "libpixman-1.la")
-        if os.path.isfile(la):
-            os.unlink(la)
-        self.copy(os.path.join(self._folder, 'COPYING'), dst='licenses')
-        tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
+            tools.rmdir(os.path.join(self.package_folder, 'lib', 'pkgconfig'))
```

Comment:
While you're at it, I think you forgot to convert these to double quotes.

Reply:
I doubt it's needed, only adds additional noise